statistics/update.go

```diff
@@ -429,6 +429,7 @@ func (h *Handle) UpdateStatsByLocalFeedback(is infoschema.InfoSchema) {
 				continue
 			}
 			newCol := *col
+			fb = fb.decodeIntValues()
```

Comment:
Without this line, will there always be panic? Why do the existing test cases not cover this case?